test-fixtures/local-package-set-config.yaml

```diff
@@ -0,0 +1,13 @@
+package:
+  dependencies:
+    - console
+    - effect
+    - prelude
+  name: aaaa
+  test:
+    dependencies: []
+    main: Test.Main
+workspace:
+  extra_packages: {}
+  package_set:
+    path: local-package-set.json
```

Comment:
Do we know if relative paths like `../../file.json` work here?

Reply:
It does, but I'll add a test

Reply:
Done